config/userconfig.ini.template

```diff
@@ -14,3 +14,7 @@ lib_path    : /path/to/dir
 [gsl]
 header_path : /path/to/dir
 lib_path    : /path/to/dir
+
+[python]
```

Comment:
Im not sure we need this, without python they can't run this script or `scons`. And we can get the python headers from `distutils.sysconfig` like you did for the libraries